deepchem/models/wandblogger.py

```diff
@@ -17,9 +17,28 @@ class WandbLogger(object):
     will log the specified metrics calculated on the specific datasets
     to the user's W&B dashboard.
 
-    If a WandbLogger is provided to the wandb_logger flag,
+    If a WandbLogger is provided to the wandb_logger flag of the model,
     the metrics are logged to Weights & Biases, along with other information
     such as epoch number, losses, sample counts, and model configuration data.
+
+    Usage
+    --------
+    Here's how WandbLogger can be used with KerasModel training and validation.
+    Apart from initialization, you do not need to call any other methods.
+    All other methods (such as setup, log_data, update_config) are part of the
+    logging API and are already integrated in the KerasModel/TorchModel classes.
+
+    >>> from deepchem.models import KerasModel, ValidationCallback
```

Comment:
Does this pass doctest? (See https://github.com/deepchem/deepchem/issues/2640 for a guide to the CI)

Reply:
My bad, must've missed it. Will look into it.

Reply:
It should be good now. If I'm reading it correctly, there's still some doctest failing but I don't think its Wandblogger related